partcad/src/partcad/cache_shape.py

```
@@ -0,0 +1,130 @@
+from .cache import Cache
+from .cache_hash import CacheHash
+from .shape import Shape
+from .user_config import user_config
+from .utils import total_size
+
+SERIALIZATION_PICKLE = 1
+SERIALIZATION_BREP = 2
+
+
+class ShapeCache(Cache):
+    def __init__(self, serialization: int = SERIALIZATION_PICKLE):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)